hackathon_site/event/models.py

```diff
@@ -28,7 +28,9 @@ class Profile(models.Model):
     STATUS_CHOICES = [("Accepted", "Accepted"), ("Waitlisted", "Waitlisted")]
 
     user = models.OneToOneField(User, on_delete=models.CASCADE)
-    team = models.ForeignKey(Team, on_delete=models.CASCADE, null=False)
+    team = models.ForeignKey(
```

Comment:
Shouldn't team also be 1:1

Reply:
No - team to profile is one to many (many people on a team). `OneToOneField` is just a foreign key field with a unique constraint.

Reply:
My bad I meant the team in registration/models.py

Reply:
No, it's the same thing. Multiple users can end up on the same team for applications (everyone is given a deafult one, but they are free to leave it and join another).